git-play-generator/src/git-play-strings.js

```diff
@@ -0,0 +1,9 @@
+const quote = "Keep it simple"
+
+console.log(quote)
+
+const fix = () => {
+    return "Fixed the issues"
+}
+
+fix()
```

Comment:
Please export the quote and fix and use them on the `bootstrap.js` file.

Reply:
Ill get on this now